### DISCUSSION TITLE
Improve the Public README.md

### DISCUSSION THREAD

**Discussion by priyankarpal** [AccessibleForAll/Support#344] [Ideas]
hey maintainers,

We can add more cool stuff here. I saw that it's just a simple  README.md. We can improve it by adding a BANNER and CONTRIBUTING guide etc.

![image](https://user-images.githubusercontent.com/88102392/221939498-0922e3c1-4024-480b-bdcf-6597c61b56a8.png)

**Comment by Vijaykv5** [comment]
Yes The Readme.md looks so Conjusted and not Great to see them very interactive. as @priyankarpal  Said, I Think its good to add a Contributor guide to it

**Comment by EmmaDawsonDev** [comment]
A banner is a great idea. Each individual repo has a contributing guide, I don’t know if it’s possible to have a guide at the top level of the org too but if it is then we can surely add one and link to it.

**Reply by priyankarpal** [reply]
Then have to add a banner.

**Reply by voldin912** [reply]
I think it looks too simple. Have to become modern and seamless. 
And there aren't enough repositories against number of members. Let's work together and make community become rich.

**Reply by EmmaDawsonDev** [reply]
What suggestions do you have @Kishida624 ?

**Comment by sandipanrakshit34** [comment]
hey please give me a star on this repository and help me to achieve the badge:

https://github.com/sandipanrakshit34/sandipanrakshit34
